.eslintrc.js

```diff
@@ -1,7 +1,7 @@
 module.exports = {
 	extends: [ 'plugin:@woocommerce/eslint-plugin/recommended' ],
 	settings: {
-		'import/resolver': 'webpack',
+		'import/resolver': 'typescript',
```

Comment:
Imports with mixed js/ts were not working, this fixes that.

Reply:
@louwie17 Are you sure about that? I think the ts import resolver is built on the eslint one, but I am not clear exactly on what can be removed I'll look into this.

Reply:
Not sure, @mattsherman added this, but when he added it he also installed the `eslint-import-resolver-webpack` plugin, so I figured it could be removed if you installed a specific typescript resolver.

Reply:
to be honest, looking deeper at this, the eslint still seems a bit broken as it is. I'll see if I can improve this.

Reply:
I think this is ok now.